webapp/components/MasonryGrid/MasonryGrid.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <ds-grid
+    :min-column-width="300"
+    v-on:calculating-item-height="startCalculation"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-20T21:55:39Z" title="Tuesday, August 20th 2019, 11:55:39 pm +02:00">Aug 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,35 @@
+<template>
+  <ds-grid
+    :min-column-width="300"
+    v-on:calculating-item-height="startCalculation"
```


```suggestion
    @calculating-item-height="startCalculation"
```